src/components/sbb-train/sbb-train.spec.ts

```diff
@@ -12,7 +12,7 @@ describe('sbb-train', () => {
       <sbb-train direction-label="Driving direction" station="Bern" direction="left">
         <mock:shadow-root>
           <div class="sbb-train">
-            <span class="sbb-screenreaderonly">Train, Driving direction Bern.</span>
+            <sbb-title visually-hidden level="6">Train, Driving direction Bern.</sbb-title>
```

Comment:
Is it necessary to use an `sbb-title` here or would a h1-6 be enough?

Reply:
@kyubisation No it's not necessary, my thought was the chance that the title component is anyways loaded in a page context is very high and so there is less payload than all the rules of the screenreader only mixin. But I can easily change it if you want?